packages/nx/src/command-line/migrate.ts

```diff
@@ -158,146 +175,137 @@ export class Migrator {
         [targetPackage]: {
           version: target.version,
           addToPackageJson: target.addToPackageJson || false,
-        },
+        } as PackageJsonUpdateForPackage,
       };
     }
 
-    let migrationsJson;
+    let migrationsJson: ResolvedMigrationConfiguration;
     try {
       migrationsJson = await this.fetch(targetPackage, targetVersion);
       targetVersion = migrationsJson.version;
     } catch (e) {
-      if (e.message.indexOf('No matching version') > -1) {
+      if (e?.message?.includes('No matching version')) {
         throw new Error(
           `${e.message}\nRun migrate with --to="package1@version1,package2@version2"`
         );
       } else {
         throw e;
       }
     }
+
     const packages = this.collapsePackages(
       targetPackage,
       targetVersion,
       migrationsJson
     );
 
-    const childCalls = await Promise.all(
+    const childPackageMigrations = await Promise.all(
       Object.keys(packages)
-        .filter((r) => {
+        .filter(([packageName]) => {
```

Comment:
It doesn't look like the typing of packages changed, this shouldn't need to be destructured right?

Reply:
I had several places where I replaced `Object.keys` with `Object.entries`, perhaps I reverted it in this case and did not revert. Good catch. `packageName` in this case would probably be the first character of the key 😅